<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>R: QTL Bayesian Interval Mapping</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link rel="stylesheet" type="text/css" href="R.css">
</head><body>
<h1> QTL Bayesian Interval Mapping
<img class="toplogo" src="../../../doc/html/logo.jpg" alt="[R logo]">
</h1>
<hr>
<div align="center">
<a href="../../../doc/html/packages.html"><img src="../../../doc/html/left.jpg" alt="[Up]" width="30" height="30" border="0"></a>
<a href="../../../doc/html/index.html"><img src="../../../doc/html/up.jpg" alt="[Top]" width="30" height="30" border="0"></a>
</div><h2>Documentation for package &lsquo;qtlbim&rsquo; version 2.0.7</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
<li><a href="../demo">Code demos</a>.  Use <a href="../../utils/help/demo">demo()</a> to run them.</li>
</ul>

<h2>Help Pages</h2>


<p align="center">
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#F">F</a>
<a href="#P">P</a>
<a href="#Q">Q</a>
<a href="#S">S</a>
<a href="#V">V</a>
</p>


<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="fitqtl.html">anova.step.fitqtl</a></td>
<td>Stepwise backward elimination and anova comparison.</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="Bnapus.html">Bnapus</a></td>
<td>Cross structure for complete Brassica napus data</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="qb-internal.html">covar.mean</a></td>
<td>Internal qtlbim routines</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="fisch.html">fisch</a></td>
<td>Eight QTL Stephens and Fisch simulated data</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="plot.html">plot.qb</a></td>
<td>Diagnostics plots for Bayesian interval mapping</td></tr>
<tr><td width="25%"><a href="BayesFactor.html">plot.qb.BayesFactor</a></td>
<td>Bayesian model selection via Bayes factors.</td></tr>
<tr><td width="25%"><a href="BestPattern.html">plot.qb.BestPattern</a></td>
<td>Proximity of common genetic architecture patterns.</td></tr>
<tr><td width="25%"><a href="close.html">plot.qb.close</a></td>
<td>Measures closeness of genetic architectures to target.</td></tr>
<tr><td width="25%"><a href="confound.html">plot.qb.confound</a></td>
<td>Examine confounding of covariate with pseudomarkers.</td></tr>
<tr><td width="25%"><a href="covar.html">plot.qb.covar</a></td>
<td>Examine GxE effect of covariates on main genetic effects.</td></tr>
<tr><td width="25%"><a href="diag.html">plot.qb.diag</a></td>
<td>Marginal and model-conditional summaries of Bayesian interval mapping diagnostics</td></tr>
<tr><td width="25%"><a href="epistasis.html">plot.qb.epistasis</a></td>
<td>Density Plots for Models Showing Epistasis and GxE Interactions.</td></tr>
<tr><td width="25%"><a href="hpd.html">plot.qb.hpdone</a></td>
<td>Highest probability density (HPD) region.</td></tr>
<tr><td width="25%"><a href="loci.html">plot.qb.loci</a></td>
<td>Jittered plot of Bayesian QTL loci samples by chromosome</td></tr>
<tr><td width="25%"><a href="modes.html">plot.qb.mainmodes</a></td>
<td>Summaries of QTL modes by chromosome.</td></tr>
<tr><td width="25%"><a href="meancomp.html">plot.qb.meancomp</a></td>
<td>Examine grand mean and covariate MCMC samples.</td></tr>
<tr><td width="25%"><a href="multloci.html">plot.qb.multloci</a></td>
<td>Summaries of multiple loci on a chromosome.</td></tr>
<tr><td width="25%"><a href="pairloci.html">plot.qb.pairloci</a></td>
<td>Summaries of epistatic pairs of loci.</td></tr>
<tr><td width="25%"><a href="plot.scanone.html">plot.qb.scanone</a></td>
<td>Plot or print qb.scanone object.</td></tr>
<tr><td width="25%"><a href="plot.scantwo.html">plot.qb.scantwo</a></td>
<td>Two dimensional LOD Plot of Epistasis/joint</td></tr>
<tr><td width="25%"><a href="sliceone.html">plot.qb.sliceone</a></td>
<td>Genome Slice to detect QTL for Phenotypic Trait</td></tr>
<tr><td width="25%"><a href="slicetwo.html">plot.qb.slicetwo</a></td>
<td>Slices for epistatic pairs.</td></tr>
<tr><td width="25%"><a href="varcomp.html">plot.qb.varcomp</a></td>
<td>Variance components for Bayesian multiple QTL</td></tr>
<tr><td width="25%"><a href="plot.html">print.qb</a></td>
<td>Diagnostics plots for Bayesian interval mapping</td></tr>
<tr><td width="25%"><a href="arch.html">print.qb.arch</a></td>
<td>Build genetic architecture with chromosomes, positions and epistatic pairs.</td></tr>
<tr><td width="25%"><a href="BayesFactor.html">print.qb.BayesFactor</a></td>
<td>Bayesian model selection via Bayes factors.</td></tr>
<tr><td width="25%"><a href="confound.html">print.qb.confound</a></td>
<td>Examine confounding of covariate with pseudomarkers.</td></tr>
<tr><td width="25%"><a href="covar.html">print.qb.covar</a></td>
<td>Examine GxE effect of covariates on main genetic effects.</td></tr>
<tr><td width="25%"><a href="diag.html">print.qb.diag</a></td>
<td>Marginal and model-conditional summaries of Bayesian interval mapping diagnostics</td></tr>
<tr><td width="25%"><a href="epistasis.html">print.qb.epistasis</a></td>
<td>Density Plots for Models Showing Epistasis and GxE Interactions.</td></tr>
<tr><td width="25%"><a href="hpd.html">print.qb.hpdone</a></td>
<td>Highest probability density (HPD) region.</td></tr>
<tr><td width="25%"><a href="loci.html">print.qb.loci</a></td>
<td>Jittered plot of Bayesian QTL loci samples by chromosome</td></tr>
<tr><td width="25%"><a href="meancomp.html">print.qb.meancomp</a></td>
<td>Examine grand mean and covariate MCMC samples.</td></tr>
<tr><td width="25%"><a href="pairloci.html">print.qb.pairloci</a></td>
<td>Summaries of epistatic pairs of loci.</td></tr>
<tr><td width="25%"><a href="plot.scanone.html">print.qb.scanone</a></td>
<td>Plot or print qb.scanone object.</td></tr>
<tr><td width="25%"><a href="sliceone.html">print.qb.sliceone</a></td>
<td>Genome Slice to detect QTL for Phenotypic Trait</td></tr>
<tr><td width="25%"><a href="slicetwo.html">print.qb.slicetwo</a></td>
<td>Slices for epistatic pairs.</td></tr>
<tr><td width="25%"><a href="varcomp.html">print.qb.varcomp</a></td>
<td>Variance components for Bayesian multiple QTL</td></tr>
<tr><td width="25%"><a href="plot.scanone.html">print.summary.qb.scanone</a></td>
<td>Plot or print qb.scanone object.</td></tr>
<tr><td width="25%"><a href="summary.scanone.html">print.summary.qb.scantwo</a></td>
<td>Summary of qb.scanone or qb.scantwo object.</td></tr>
<tr><td width="25%"><a href="qb-internal.html">pull.grid</a></td>
<td>Internal qtlbim routines</td></tr>
<tr><td width="25%"><a href="qb-internal.html">pull.loci</a></td>
<td>Internal qtlbim routines</td></tr>
</table>

<h2><a name="Q">-- Q --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="arch.html">qb.arch</a></td>
<td>Build genetic architecture with chromosomes, positions and epistatic pairs.</td></tr>
<tr><td width="25%"><a href="arch.html">qb.arch.default</a></td>
<td>Build genetic architecture with chromosomes, positions and epistatic pairs.</td></tr>
<tr><td width="25%"><a href="arch.html">qb.arch.qb.BestPattern</a></td>
<td>Build genetic architecture with chromosomes, positions and epistatic pairs.</td></tr>
<tr><td width="25%"><a href="arch.html">qb.arch.step.fitqtl</a></td>
<td>Build genetic architecture with chromosomes, positions and epistatic pairs.</td></tr>
<tr><td width="25%"><a href="BayesFactor.html">qb.BayesFactor</a></td>
<td>Bayesian model selection via Bayes factors.</td></tr>
<tr><td width="25%"><a href="BestPattern.html">qb.best</a></td>
<td>Proximity of common genetic architecture patterns.</td></tr>
<tr><td width="25%"><a href="BestPattern.html">qb.BestPattern</a></td>
<td>Proximity of common genetic architecture patterns.</td></tr>
<tr><td width="25%"><a href="BayesFactor.html">qb.bf</a></td>
<td>Bayesian model selection via Bayes factors.</td></tr>
<tr><td width="25%"><a href="close.html">qb.close</a></td>
<td>Measures closeness of genetic architectures to target.</td></tr>
<tr><td width="25%"><a href="coda.html">qb.coda</a></td>
<td>Coerce to an MCMC object for use with the coda package.</td></tr>
<tr><td width="25%"><a href="confound.html">qb.confound</a></td>
<td>Examine confounding of covariate with pseudomarkers.</td></tr>
<tr><td width="25%"><a href="covar.html">qb.covar</a></td>
<td>Examine GxE effect of covariates on main genetic effects.</td></tr>
<tr><td width="25%"><a href="qb-internal.html">qb.cross</a></td>
<td>Internal qtlbim routines</td></tr>
<tr><td width="25%"><a href="qb-internal.html">qb.cross.class</a></td>
<td>Internal qtlbim routines</td></tr>
<tr><td width="25%"><a href="data.html">qb.data</a></td>
<td>Prepares data for qb.mcmc</td></tr>
<tr><td width="25%"><a href="qb-internal.html">qb.demo</a></td>
<td>Internal qtlbim routines</td></tr>
<tr><td width="25%"><a href="diag.html">qb.diag</a></td>
<td>Marginal and model-conditional summaries of Bayesian interval mapping diagnostics</td></tr>
<tr><td width="25%"><a href="modes.html">qb.epimodes</a></td>
<td>Summaries of QTL modes by chromosome.</td></tr>
<tr><td width="25%"><a href="epistasis.html">qb.epistasis</a></td>
<td>Density Plots for Models Showing Epistasis and GxE Interactions.</td></tr>
<tr><td width="25%"><a href="genoprob.html">qb.genoprob</a></td>
<td>Grid point and genotype probability computation method</td></tr>
<tr><td width="25%"><a href="qb-internal.html">qb.get</a></td>
<td>Internal qtlbim routines</td></tr>
<tr><td width="25%"><a href="hpdchr.html">qb.hpdchr</a></td>
<td>Find highest probability density (HPD) region by Chromosome.</td></tr>
<tr><td width="25%"><a href="hpd.html">qb.hpdone</a></td>
<td>Highest probability density (HPD) region.</td></tr>
<tr><td width="25%"><a href="epistasis.html">qb.intcov</a></td>
<td>Density Plots for Models Showing Epistasis and GxE Interactions.</td></tr>
<tr><td width="25%"><a href="remove.html">qb.legacy</a></td>
<td>Legacy update, remove or recover of qb object and associated MCMC samples</td></tr>
<tr><td width="25%"><a href="qb-internal.html">qb.load</a></td>
<td>Internal qtlbim routines</td></tr>
<tr><td width="25%"><a href="loci.html">qb.loci</a></td>
<td>Jittered plot of Bayesian QTL loci samples by chromosome</td></tr>
<tr><td width="25%"><a href="modes.html">qb.mainmodes</a></td>
<td>Summaries of QTL modes by chromosome.</td></tr>
<tr><td width="25%"><a href="mcmc.html">qb.mcmc</a></td>
<td>Bayesian Mutiple Interacting QTL mapping using MCMC</td></tr>
<tr><td width="25%"><a href="meancomp.html">qb.meancomp</a></td>
<td>Examine grand mean and covariate MCMC samples.</td></tr>
<tr><td width="25%"><a href="model.html">qb.model</a></td>
<td>Set up interacting QTL model for qb.mcmc</td></tr>
<tr><td width="25%"><a href="multloci.html">qb.multloci</a></td>
<td>Summaries of multiple loci on a chromosome.</td></tr>
<tr><td width="25%"><a href="pairloci.html">qb.pairloci</a></td>
<td>Summaries of epistatic pairs of loci.</td></tr>
<tr><td width="25%"><a href="remove.html">qb.recover</a></td>
<td>Legacy update, remove or recover of qb object and associated MCMC samples</td></tr>
<tr><td width="25%"><a href="remove.html">qb.remove</a></td>
<td>Legacy update, remove or recover of qb object and associated MCMC samples</td></tr>
<tr><td width="25%"><a href="qb-internal.html">qb.reorder</a></td>
<td>Internal qtlbim routines</td></tr>
<tr><td width="25%"><a href="qb-internal.html">qb.save</a></td>
<td>Internal qtlbim routines</td></tr>
<tr><td width="25%"><a href="scanone.html">qb.scanone</a></td>
<td>Genome Scan for Main Loci Involved in Phenotypic Trait</td></tr>
<tr><td width="25%"><a href="scantwo.html">qb.scantwo</a></td>
<td>Genome Scan for Pairs of Loci Involved in Phenotypic Trait</td></tr>
<tr><td width="25%"><a href="sim.html">qb.sim.cross</a></td>
<td>Simulates QTL related data for an F2 or BC cross.</td></tr>
<tr><td width="25%"><a href="sliceone.html">qb.sliceone</a></td>
<td>Genome Slice to detect QTL for Phenotypic Trait</td></tr>
<tr><td width="25%"><a href="slicetwo.html">qb.slicetwo</a></td>
<td>Slices for epistatic pairs.</td></tr>
<tr><td width="25%"><a href="modes.html">qb.split.chr</a></td>
<td>Summaries of QTL modes by chromosome.</td></tr>
<tr><td width="25%"><a href="sweave.html">qb.sweave</a></td>
<td>Run sweave to automate QTL search with MCMC samples.</td></tr>
<tr><td width="25%"><a href="varcomp.html">qb.varcomp</a></td>
<td>Variance components for Bayesian multiple QTL</td></tr>
<tr><td width="25%"><a href="qbExample.html">qbExample</a></td>
<td>MCMC sample for fake BC example</td></tr>
<tr><td width="25%"><a href="qbHyper.html">qbHyper</a></td>
<td>MCMC samples for hyper dataset from R/qtl</td></tr>
<tr><td width="25%"><a href="qbSim.html">qbSim</a></td>
<td>MCMC samples used in scan.pdf document</td></tr>
<tr><td width="25%"><a href="qbSim.html">qbSimEpi</a></td>
<td>MCMC samples used in scan.pdf document</td></tr>
<tr><td width="25%"><a href="qbSim.html">qbSimMain</a></td>
<td>MCMC samples used in scan.pdf document</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="sim.html">sim.data</a></td>
<td>Simulates QTL related data for an F2 or BC cross.</td></tr>
<tr><td width="25%"><a href="fitqtl.html">step.fitqtl</a></td>
<td>Stepwise backward elimination and anova comparison.</td></tr>
<tr><td width="25%"><a href="subset.html">subset.qb</a></td>
<td>Subsetting Bayesian interval mapping data</td></tr>
<tr><td width="25%"><a href="plot.html">summary.qb</a></td>
<td>Diagnostics plots for Bayesian interval mapping</td></tr>
<tr><td width="25%"><a href="arch.html">summary.qb.arch</a></td>
<td>Build genetic architecture with chromosomes, positions and epistatic pairs.</td></tr>
<tr><td width="25%"><a href="BayesFactor.html">summary.qb.BayesFactor</a></td>
<td>Bayesian model selection via Bayes factors.</td></tr>
<tr><td width="25%"><a href="BestPattern.html">summary.qb.BestPattern</a></td>
<td>Proximity of common genetic architecture patterns.</td></tr>
<tr><td width="25%"><a href="close.html">summary.qb.close</a></td>
<td>Measures closeness of genetic architectures to target.</td></tr>
<tr><td width="25%"><a href="confound.html">summary.qb.confound</a></td>
<td>Examine confounding of covariate with pseudomarkers.</td></tr>
<tr><td width="25%"><a href="covar.html">summary.qb.covar</a></td>
<td>Examine GxE effect of covariates on main genetic effects.</td></tr>
<tr><td width="25%"><a href="diag.html">summary.qb.diag</a></td>
<td>Marginal and model-conditional summaries of Bayesian interval mapping diagnostics</td></tr>
<tr><td width="25%"><a href="epistasis.html">summary.qb.epistasis</a></td>
<td>Density Plots for Models Showing Epistasis and GxE Interactions.</td></tr>
<tr><td width="25%"><a href="hpd.html">summary.qb.hpdone</a></td>
<td>Highest probability density (HPD) region.</td></tr>
<tr><td width="25%"><a href="loci.html">summary.qb.loci</a></td>
<td>Jittered plot of Bayesian QTL loci samples by chromosome</td></tr>
<tr><td width="25%"><a href="modes.html">summary.qb.mainmodes</a></td>
<td>Summaries of QTL modes by chromosome.</td></tr>
<tr><td width="25%"><a href="meancomp.html">summary.qb.meancomp</a></td>
<td>Examine grand mean and covariate MCMC samples.</td></tr>
<tr><td width="25%"><a href="multloci.html">summary.qb.multloci</a></td>
<td>Summaries of multiple loci on a chromosome.</td></tr>
<tr><td width="25%"><a href="pairloci.html">summary.qb.pairloci</a></td>
<td>Summaries of epistatic pairs of loci.</td></tr>
<tr><td width="25%"><a href="summary.scanone.html">summary.qb.scanone</a></td>
<td>Summary of qb.scanone or qb.scantwo object.</td></tr>
<tr><td width="25%"><a href="summary.scanone.html">summary.qb.scantwo</a></td>
<td>Summary of qb.scanone or qb.scantwo object.</td></tr>
<tr><td width="25%"><a href="sim.html">summary.qb.sim</a></td>
<td>Simulates QTL related data for an F2 or BC cross.</td></tr>
<tr><td width="25%"><a href="sliceone.html">summary.qb.sliceone</a></td>
<td>Genome Slice to detect QTL for Phenotypic Trait</td></tr>
<tr><td width="25%"><a href="slicetwo.html">summary.qb.slicetwo</a></td>
<td>Slices for epistatic pairs.</td></tr>
<tr><td width="25%"><a href="varcomp.html">summary.qb.varcomp</a></td>
<td>Variance components for Bayesian multiple QTL</td></tr>
</table>

<h2><a name="V">-- V --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="vern.html">vern</a></td>
<td>Eight week vernalization data for Brassica napus</td></tr>
</table>
</body></html>
